packages/component/src/Activity/private/Feedback.tsx

```diff
@@ -41,14 +43,31 @@ const Feedback = memo(({ actions }: Props) => {
     return () => clearTimeout(timeout);
   }, [clearTimeout, postActivityRef, selectedAction, setTimeout]);
 
+  const actionProps = useMemo(
+    () =>
+      [...actions].some(action => action.actionStatus === 'CompletedActionStatus')
+        ? {
+            title: localize('VOTE_COMPLETE_ALT'),
+            disabled: true
+          }
```

Comment:
Later, should we allow changing mind after 500ms?

Reply:
I think so, though converging previous Web Chat and new Fluent requirement was already tough